### PR TITLE
Mark occurence item in string with a special property

### LINKED ISSUE #12780: 'occurrences' Returns occurrences inside strings
From https://github.com/Microsoft/vscode/issues/16909

**TypeScript Version:**  2.1.4 

**Code**

```ts
"fdsafdas";
"fdsafdas";
```

**Expected behavior:**
In TS 2.0.10, when calling tsserver 'occurrences' for any position inside `"fdsafdas"`, we get no results back.

**Actual behavior:**
In TS 2.1.4, we now get back occurrences of the string throughout the file.

### PR DESCRIPTION
**Pull request by zhengbli**
Partially fixes #12780

### CODE REVIEW

File: src/server/session.ts [hunk 1]
```diff
@@ -651,16 +651,21 @@ namespace ts.server {
...
                 const scriptInfo = project.getScriptInfo(fileName);
                 const start = scriptInfo.positionToLineOffset(textSpan.start);
                 const end = scriptInfo.positionToLineOffset(ts.textSpanEnd(textSpan));
-                return {
+                const result: protocol.OccurrencesResponseItem = {
                     start,
                     end,
                     file: fileName,
-                    isWriteAccess,
+                    isWriteAccess
```
Comment (alfaproject) [inline src/server/session.ts]: Don't remove the trailing comma (;

Comment (zhengbli) [inline src/server/session.ts]: Reason being?

Comment (mhegazy) [inline src/server/session.ts]: the diff only shows the changed property, and next time when someone adds/removes a property same occurs.
that happens to be the main reason why ES2017 supports trailing commas in argument lists, see https://github.com/tc39/proposal-trailing-function-commas

Comment (alfaproject) [inline src/server/session.ts]: Exactly. Git attribution, less lines changes, less conflicts (;

Comment (zhengbli) [inline src/server/session.ts]: Ha that does make sense. Thanks for pointing out! will update.

**Pull request merged**